### DISCUSSION TITLE
Using the TeX formatter

### DISCUSSION THREAD

**Discussion by ralfwimmer** [rouge-ruby/rouge#1817] [Q&A]
Hi all,

I am trying to use Rouge for formating code to be used in a LaTeX document. This yields something like

\begin{RG*}
...
\end{RG*}

Which LaTeX package do I have to load such that LaTeX understands the commands and environments that are emited by Rouge?

Best
Ralf.

**Comment by jneen** [comment]
Hi! This is as far as I can tell a fairly standard way of doing it. You are free to implement the RG* environment in any way you like.

I wrote the below package for my thesis:

https://github.com/jneen/hilite.tex
